envoy/WORKSPACE

```diff
@@ -7,7 +7,7 @@ workspace(name = "cilium")
 #
 # No other line in this file may have ENVOY_SHA followed by an equals sign!
 #
-ENVOY_SHA = "12434bee6a6be56a62955999ed38e16a7a9ee054"
+ENVOY_SHA = "fc3693b5205518171a590d87e51ae317040f9276"
```

Comment:
Please pull the changes on the Envoy side out of this PR. Those have nothing to do with logging.

Reply:
"Envoy rebase is needed to get support for the new '--log-format' command
line option."

Reply:
Yep, my bad.